specs/vendor_spec.rb

```diff
@@ -0,0 +1,2 @@
+require_relative './spec_helper'
+require_relative '../far_mar'
```

Comment:
You can do this line in the `spec_helper` and not have to do it in all the individual specs.